src/tag/main.scss

```diff
@@ -121,6 +121,14 @@
     &tag-closable {
         position: relative;
 
+        &.#{$css-prefix}tag-medium > .#{$css-prefix}tag-body {
+            max-width: calc(100% - #{$tag-size-m-height});
+        }
+
+        &.#{$css-prefix}tag-small > .#{$css-prefix}tag-body {
+            max-width: calc(100% - #{$tag-size-s-height});
+        }
+
```

Comment:
why is `-tag-large` left?

Reply:
this wasn't necessary for select, since it doesn't use large tags, but i can add it because we don't know if it will need in the future